rhscl-postgresql-apb/roles/rhscl-postgresql-apb-openshift/README

```diff
@@ -0,0 +1,41 @@
+Postgresql-Demo-APB-Openshift
```

Comment:
Will there be a README for every apb? Would a general doc on creating and using APBs be more effective?